src/main/java/bio/terra/common/stairway/TracingHook.java

```diff
@@ -92,7 +92,7 @@ public static void storeCurrentContextAsSubmission(FlightMap inputMap) {
    * <p>Note that the same Flight may be run multiple times by Stairway.
    */
   private class TraceFlightHook implements DynamicHook {
-    private Scope flightScope = null;
+    private Scope flightScope;
```

Comment:
no action required: go/java-practices/redundancy#default-initialization-for-non-final-fields :)